packages.yml

```diff
@@ -1,3 +1,6 @@
 packages:
-- package: fivetran/linkedin_source
-  version: [">=0.6.0", "<0.7.0"]
\ No newline at end of file
+# - package: fivetran/linkedin_source
+#   version: [">=0.7.0", "<0.8.0"]
```

Comment:
We should also update the README to reference this new version range of the source package.

Reply:
Reminder to update before merge